inputs_land.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: index
+title: External Inputs to the Land Model
+prev: diagram.html
+next: land.html
+gcam-version: v5.3 
+---
+
+GCAM's external land inputs include information on land, carbon, other emissions, and the value of unmanaged land in the historical period. These inputs are required for each [global land unit](common_assumptions.html#global-land-units) and [historical year](common_assumptions.html#historical-years).
+
+
+## External Inputs
+
+### Description
+
+| Name | Description | Type | Source | Resolution | Unit |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| Historical Land Use and Land Cover | Land area by region, land type and year. Land cover data is provided beginning in 1700 in order to spin-up the carbon cycle within GCAM. Crop-specific harvested area is provided only for a single year to downscale FAO data to a subnational level. Similarly, the division between irrigated and rainfed land is provided for a single year only. | External Data Set | <a href="https://github.com/JGCRI/moirai">Moirai</a> | Raw data is by country, water basin and land type. | thousand $$km^2$$ |
```

Comment:
Maybe note that the downscaling data is only provided for one year because it's only available for one year (for irr vs rainfed at least).